Package.swift

```diff
@@ -1,14 +1,43 @@
+// swift-tools-version:5.1
+//
```

Comment:
Very minor, but would you mind removing this copyright header?

Reply:
Sure, removed.